link_shortener/core/filter.py

```diff
@@ -0,0 +1,23 @@
+'''
+Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
+Licensed under the MIT (Expat) License (see LICENSE in Documentation).
+'''
+
+
+def filter_links(data, filters):
+    owners = set(row['owner'] for row in data)
+
+    if filters['owner']:
+        data = [row for row in data if row['owner'] == filters['owner']]
+
+    if filters['search']:
+        data = [row for row in data if filters['search'] in row['endpoint']]
+
+    return {"owners": owners, "data": data}
+
+
+def define_active(filters):
+    if filters['is_active'] == 'false':
+        return False
+    else:
```

Comment:
Tenhle else tady nepotřebuješ. Ničemu nepřekáží, to jen že spousta lidí vůbec else nepíšou když nemusí tak aby tě to nemátlo někde pak

Reply:
Tenhle return tu vlastně stejně nebude když sem přesuneš ty dva řádky navíc. Ale stejně to byla jen poznámka

Reply:
Takto by to malo byt, ze?
```
def define_active(filters):
    if filters['is_active'] == 'false':
        return False
    
    return True
```